app/src/main/java/com/example/snookerapi/data/api/SnookerApi.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.snookerapi.data.api
+
+import com.example.snookerapi.data.model.EventDTO
+import com.example.snookerapi.data.model.EventProfileDTO
+import com.example.snookerapi.data.model.PlayerDTO
+import com.example.snookerapi.data.model.PlayerProfileDTO
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface SnookerApi {
+
+    @GET("?t=10&st=p")
+    suspend fun getPlayers(
+        @Query("s") s: Int,
+    ): List<PlayerDTO>
+
+    @GET(".")
+    suspend fun getPlayerProfile(
+        @Query("p") id: Int,
+    ): List<PlayerProfileDTO>
+
+    @GET("?t=5")
```

Comment:
я помню у тебя в апи не все так очевидно по запросам, такие штуки лучше выносить в константы
```suggestion
    @GET("?t=$REQUEST_ID_EVENTS")
```

Reply:
Done